### DISCUSSION TITLE
GITHUB STUDENT REVERIFICATION

### DISCUSSION THREAD

**Discussion by such3** [community/community#130509] [GitHub Education]
### Hi everyone,

Fellow developers,

I'm reaching out with a pressing issue and hope someone can offer guidance. My GitHub Student Developer Pack is expiring in two weeks, and I'm struggling to re verify my account. The problem is that GitHub requires a college email address, which my institution refuses to provide.

I'm willing to provide alternative documents to prove my enrollment, but the verification process won't let me proceed.

Has anyone faced a similar issue? Are there any workarounds or suggestions on how to overcome this hurdle? I'd greatly appreciate any advice or assistance.

**Comment by TheRealFREDP3D** [comment]
You should refer to this document to learn your other options:

[Who is eligible Faq](https://github.com/orgs/community/discussions/111352#who-is-eligible-for-the-pack)

**Comment by Rishi-Sudhakar** [comment]
Yes you can use other documents,
They accept the following proof types, in order of preference:

1. Student ID
2. Transcript
3. Enrollment or affiliation letter
4. Class schedule
Your first and last name, complete school name, and a date demonstrating current academic affiliation must appear in your document.

If you select your institution and it requires you to enter institution email, then, choose,
"My selected school has incorrect or incomplete information e.g. domains or campus location".
